### ISSUE TITLE
Simple Masonry: Change Item label to image Title if available

### ISSUE THREAD

**Issue by Misplon** [siteorigin/so-widgets-bundle#924]
Currently, it's hard re-ordering the Simple Masonry and similar widgets because all you see is:

<img width="490" alt="Edit_Page_‹_SiteOrigin_Premium_Demo_—_WordPress" src="https://user-images.githubusercontent.com/789159/70155573-2c879580-16bb-11ea-9500-9e8ca24f30fe.png">

Let's change "Item" to the image Title tag if available.

**Comment by AlexGStapleton** [comment]
I just went to do this and it appears to have already in place. Can you please try setting a Title, saving and then checking if a title has been set?

![](https://i.imgur.com/JFJrrst.png)

**Comment by Misplon** [comment]
Thanks, you're right. Perhaps I was logged this for the incorrect widget, not sure. Closing.

**Comment by aequitascs** [comment]
Hi guys. I've already sunk quite a lot of time into resolving this, so I'd like to request that this is re-opened.

When I tested it, I took a literal interpretation of the original request. The "title" field in each masonry element is different from the _image_ title that is associated with the image in the media library. As the code stands at the moment, the masonry items will all appear as "Item" if the title field is not populated.

However, I've made a change (I'm about to check it in and generate a PR) that actually uses the image's title setting in Wordpress is the title is not set in the masonry element form. By default in Wordpress, this is the image file name. The result is this:

![image](https://user-images.githubusercontent.com/23128430/76241495-be181d80-623d-11ea-99d3-fdb68abcb448.png)

In this example, all titles like "Test X" are from the "Title" field, and all others come directly from the Wordpress media title associated with the image.

So, I think this might be an improvement that addresses the original problem, which will still occur when masonry images don't have an explicit title set in their own form.
